src/mqtt/qmqtt_client.h

```diff
@@ -67,6 +67,16 @@ enum ConnectionState
     STATE_CONNECTED,
     STATE_DISCONNECTED
 };
+namespace MqttError {
+    enum MqttError {
```

Comment:
This is an internal enum, and should not be part of the public interface. Move it to qmqtt_client_p.cpp where it is used.
The name of the first value of enum MqttError is incorrect (the value indicates that the connection is OK). It is not used as well, so remove it. This will also remove the need to the MqttError namespace.